xblock/exceptions.py

```diff
@@ -30,8 +30,9 @@ def __init__(self, saved_fields, dirty_fields):
         `dirty_fields` - a set of fields that were left dirty after the save
         """
         # Exception is an old-style class, so can't use super
-        Exception.__init__(self)
+        Exception.__init__(self, message)
```

Comment:
Still unclear, please see this.
[xblock/exceptions.py](https://github.com/edx/XBlock/blob/111935cab69a9128948180e6b9673c391d2ff165/xblock/exceptions.py#L17)